src/features/cards/ui/editCard/editCardModal/EditCardModal.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryObj } from '@storybook/react'
+
+import { EditCardModal } from './EditCardModal'
+
+import { EditIcon } from '@/assets'
+import { IconButton, OptionType } from '@/components'
+
+const meta: Meta<typeof EditCardModal> = {
+  title: 'Cards/EditCardModal',
+  component: EditCardModal,
+  tags: ['autodocs'],
+}
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+const options: OptionType[] = [
+  { value: 'text', title: 'Text' },
+  { value: 'picture', title: 'Picture' },
+]
+
+export const DefaultEditCardModal: Story = {
+  args: {
+    trigger: <IconButton icon={<EditIcon size={1.6} />} />,
```

Comment:
pass prop size to the IconButton component (not EditIcon)

Reply:
Done!